test/groovy/MtaBuildTest.groovy

```diff
@@ -61,7 +61,7 @@ public class MtaBuildTest extends BasePiperTest {
 
         stepRule.step.mtaBuild(script: nullScript, buildTarget: 'NEO')
 
-        assert shellRule.shell.find { c -> c.contains('PATH=./node_modules/.bin:/usr/bin')}
+        assert shellRule.shell.find { c -> c.contains('PATH=./node_modules/.bin:$PATH')}
```

Comment:
Please remove line 55 `binding.setVariable('PATH', '/usr/bin')` it is not needed anymore.